src/dune_engine/display.ml

```diff
@@ -1,47 +1,9 @@
-open Import
-
-type verbosity =
+type t =
```

Comment:
Should this type move too, or is it hard for some reason? (Maybe switching to polymorphic variants might help disentangle things.)

Reply:
There's a comment in the .mli. We'll be able to move this type once we separate running actions from printing the output to the console.